mantidproject/mantid
# ResNorm "plot result: fit" plots only data

Fixes #13594

When using the Plot option: Fit in ResNorm, this should now plot both the original spectrum data, and the calculated fit.
# To Test
- Open ResNorm interface (Interfaces > Indirect > Bayes > ResNorm)
- Load in relevant Vanadium and Resolution data
- Run ResNorm **with the plot Option set to Fit**
- Ensure that a plot of the original data **AND** the calculated fit is produced when the algorithm concludes
